src/Symfony/Component/Translation/TranslatorBagInterface.php

```diff
@@ -24,10 +24,13 @@ interface TranslatorBagInterface
      * Gets the catalogue by locale.
      *
      * @param string|null $locale The locale or null to use the default
-     *
      * @return MessageCatalogueInterface
      *
      * @throws InvalidArgumentException If the locale contains invalid characters
      */
     public function getCatalogue(string $locale = null);
+
+    public function getCatalogues(): array;
+
+    public function getDomains(): array;
```

Comment:
in order to not break BC, these new method should be declared using `@method` annotationinstead